### ISSUE TITLE
@babel/preset-env cannot be found

### ISSUE THREAD

**Issue by palmerj3** [babel/babel#6593]
Choose one: is this a bug report or feature request?

Bug

### Expected Behavior
`yarn add "@babel/preset-env"
`
Would work.

### Current Behavior
```
yarn add --dev "@babel/preset-env"
yarn add v1.2.1
warning package.json: License should be a valid SPDX license expression
warning foo@1.0.0: License should be a valid SPDX license expression
[1/4] 🔍  Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@babel%2fpreset-env: Not found".
info If you think this is a bug, please open a bug report with the information provided in "/Users/jpalmer/Development/foo/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

### Possible Solution
The [README](https://github.com/babel/babel/blob/master/experimental/babel-preset-env/README.md) I think was updated before the package was published.

**Comment by existentialism** [comment]
@palmerj3 yes we updated all the docs in preparation for our next release, which unfortunately got pushed back slightly. we intend on pushing the next beta (first with scoped pacakges) asap.

**Comment by palmerj3** [comment]
No worries at all - I love and respect all the work you all do for this project!

Just wanted to flag it as an issue since I was installing babel-preset-env for a new project and noticed the docs were off.

**Comment by tomasdev** [comment]
May be related due to release conflicts
package json:
```
"devDependencies": {
    "@babel/core": "^7.0.0-beta.5",
    "@babel/preset-env": "^7.0.0-beta.4",
```
After `npm install` it throws

`npm WARN @babel/[blablabla-module-name]@7.0.0-beta.4 requires a peer of @babel/core@7.0.0-beta.3 but none was installed.` even though core got installed.

**Comment by loganfsmyth** [comment]
@tomasdev If you're using the beta, you should be 100%  making sure you have the same version of every package. Using `^` on the betas can easily make breaking changes. If you change them both to `beta.5` you shouldn't see that warning.

**Comment by tomasdev** [comment]
Yeah, ended up rolling back to `babel-preset-env` until the `@babel/xxx` things get out of beta.

**Comment by loganfsmyth** [comment]
fair, as long as you're not using a mix of 6.x and 7.x, or different versions of 7.x-beta, you should be good.

**Comment by hzoo** [comment]
Published `@babel/preset-env` and others in `7.0.0-beta.31`l but yeah should use exact versions in beta
